File: WEEK 1/C++/Coin Value Calculation.cpp
#include <iostream>
using namespace std;

int main() {
    int quarters, dimes, nickels;

    cout << "Enter the number of quarters: ";
    cin >> quarters;

    cout << "Enter the number of dimes: ";
    cin >> dimes;

    cout << "Enter the number of nickels: ";
    cin >> nickels;

    int totalValueInPennies = (quarters * 25) + (dimes * 10) + (nickels * 5);

    printf("Total value in pennies: %d\n", totalValueInPennies);

    return 0;
}

File: WEEK 1/java/Coin Value Calculation.java
import java.util.Scanner;

public class CoinValueCalculator {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        // Prompt the user for the number of quarters, dimes, and nickels
        System.out.print("Enter the number of quarters: ");
        int quarters = input.nextInt();

        System.out.print("Enter the number of dimes: ");
        int dimes = input.nextInt();

        System.out.print("Enter the number of nickels: ");
        int nickels = input.nextInt();

        // Calculate the total value of the coins in pennies
        int totalValueInPennies = (quarters * 25) + (dimes * 10) + (nickels * 5);

        System.out.println("Total value of the coins: " + totalValueInPennies + " pennies");

        input.close();
    }
}
